ndb_api/time_machine.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
I understand we are using the time-machine API endpoint here but the APIs calls themselves are related to the clone functionality. should we rename this file to clone.go?

Reply:
While I understand that naming it clone.go would make more sense, but I still feel we should keep this under TimeMachine to stay coherent with the rest API explorer on NDB. Thoughts?

Reply:
Yeah I think it might be confusing for maintaining purposes if our file conventions are not consistent with NDB apis even though its not as logical.

Reply:
but the ndb operator developer doesn't know that :).
its an open source project and the reader of code will not understand this background. They may not be familiar with NDB's TM concept either.

Reply:
Okay, that makes sense! Will move into clone.go and associated files.